hoomd/md/methods/rattle.py

```diff
@@ -448,6 +448,138 @@ def _attach(self):
         self._cpp_obj = my_class(sim.state._cpp_sys_def,
                                  sim.state._get_group(self.filter),
                                  self.manifold_constraint._cpp_obj, self.kT,
+                                 False, False, self.tolerance)
+
+        # Attach param_dict and typeparam_dict
+        super()._attach()
+
+
+class OverdampedViscous(MethodRATTLE):
+    r"""Overdamped viscous dynamics with RATTLE constraint.
+
+    Args:
+        filter (`hoomd.filter.ParticleFilter`): Subset of particles to
+            apply this method to.
+
+        manifold_constraint (:py:mod:`hoomd.md.manifold.Manifold`): Manifold
+            constraint.
+
+        alpha (`float`): When set, use :math:`\alpha d_i` for the
+            drag coefficient where :math:`d_i` is particle diameter.
+            Defaults to None
+            :math:`[\mathrm{mass} \cdot \mathrm{length}^{-1}
+            \cdot \mathrm{time}^{-1}]`.
+
+        tolerance (`float`): Defines the tolerated error particles are allowed
+            to deviate from the manifold in terms of the implicit function.
+            The units of tolerance match that of the selected manifold's
+            implicit function. Defaults to 1e-6
+
+    `OverdampedViscous` uses the same integrator as
+    `hoomd.md.methods.OverdampedViscous`, with the additional force term
+    :math:`- \lambda \vec{F}_\mathrm{M}`. The force
+    :math:`\vec{F}_\mathrm{M}` keeps the particles on the manifold constraint,
+    where the Lagrange multiplier :math:`\lambda` is calculated via the RATTLE
+    algorithm. For more details about overdamped viscous dynamics see
+    `hoomd.md.methods.OverdampedViscous`.
+
+    Examples of using ``manifold_constraint``::
+
+        sphere = hoomd.md.manifold.Sphere(r=10)
+        odv_rattle = hoomd.md.methods.rattle.OverdampedViscous(
+        filter=hoomd.filter.All(), manifold_constraint=sphere,
+        seed=1, alpha=1.0)
+        integrator = hoomd.md.Integrator(dt=0.001, methods=[odv_rattle],
+        forces=[lj])
```

Comment:
```suggestion
        odv_rattle = hoomd.md.methods.rattle.OverdampedViscous(
            filter=hoomd.filter.All(), manifold_constraint=sphere, seed=1,
            alpha=1.0)
        integrator = hoomd.md.Integrator(
            dt=0.001, methods=[odv_rattle], forces=[lj])
```
Make the line splits more readable.